### ISSUE TITLE
Inconsistent spacing with button groups across multiple browsers

### ISSUE THREAD

**Issue by EasyAs123** [zurb/foundation#6613]
This is a modification and re-opening of #5735, which was closed despite not being fixed.

The issue looks like this (from the Buttons Group documentation):
![Space issue](https://cloud.githubusercontent.com/assets/5585636/4139125/61f66762-3397-11e4-8cc5-230ff6d88e8c.png)
(credit for image goes to ghost who opened #5735)

Notice the uneven gap on the left.

I made this issue because I found a solution that has worked for me and I wanted to post my findings hopes that someone might implement it as a more permanent solution to this issue. I do not have enough familiarity with the project to feel comfortable messing with its code, so I am submitting it for scrutiny as an issue. As far as I can tell it fixes the issue across all web-browsers and does not break anything, but your mileage may vary.

Here is the solution:

in \foundation\components\_button-groups.scss,  near line 152 :

change

    // We use this to make the buttons even width across their container
    @if $even {
        width: percentage((100/$even) / 100);
        button, .button { width: 100%; }
    }
to

    // We use this to make the buttons even width across their container
    @if $even {
        width: percentage((100/$even) / 100);
        button, .button { width: 101%; }
    }

Basically just change width from 100% to 101%.

It doesn't seem to have broken anything and as far as I can tell it fixes the problem across all my browsers!

Here it is afterwards, fixed, in one of my own pages
![fix](https://cloud.githubusercontent.com/assets/12797331/8038196/5da23d80-0e02-11e5-8ccb-681a35edb29a.png)

Cheers!

**Comment by rafibomb** [comment]
Thanks for posting your solution. I don't think it's ideal though as a framework solution. This spacing is due to how Chrome is rounding widths. 

I dug back in the codebase and found the commit that fixed this a while back. https://github.com/zurb/foundation/commit/d13e7e3b453b572317cf966e9659613d212519aa
Looks like the mixin has changed a bit since then but we just added the fix back in.

**Comment by rafibomb** [comment]
55ddbb7
